app/src/main/java/com/yatochk/secure/app/ui/calculator/CalculatorActivity.kt

```diff
@@ -20,6 +20,9 @@ class CalculatorActivity : BaseActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_calculator)
+        keyboard.setKeysLongListener {
```

Comment:
no need KeysLongListener for keyboard. try invoke KeysListener with Key.KEY_LONG_DELETE